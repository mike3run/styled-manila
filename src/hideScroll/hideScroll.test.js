import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { hideScroll } from '..'

test('hideScroll works!', () => {
  const Test = styled.div`
    ${hideScroll}
  `
  const tree = renderer.create(<Test />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toHaveStyleRule('display', 'none', {
    modifier: '::-webkit-scrollbar'
  })
})
