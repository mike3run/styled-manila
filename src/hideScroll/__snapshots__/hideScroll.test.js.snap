// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hideScroll works! 1`] = `
.c0::-webkit-scrollbar {
  display: none;
}

<div
  className="c0"
/>
`;
